docs/fendermint/running.md

```diff
@@ -375,6 +375,7 @@ configuration will look for it at `~/.fendermint/bundle.car`, so we might as wel
 ```shell
 make actor-bundle
```

Comment:
Nice, but then it looks like [this](https://github.com/consensus-shipyard/ipc/blob/6c83a7c2ec549ae4dfac11415372827323b2507e/fendermint/Makefile#L101) should also depend on the `$(ACTORS_BUNDLE)`.